cdk/cmr-stac-dev/bin/cmr-stac-dev.ts

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+import * as cdk from 'aws-cdk-lib';
+import { CmrStacStack } from '../lib/cmr-stac-dev-stack';
+
+const {
+  AWS_ACCOUNT = '1234567890',
+  AWS_REGION = 'us-east-1',
+  STAGE_NAME = 'dev',
+} = process.env;
+const app = new cdk.App();
+new CmrStacStack(app, `cmr-stac-${STAGE_NAME}`, {
```

Comment:
I originally started having just one cdk stack I think we could still do that it'd just require some conditional logic into the main one but, it seemed better to just have 2 open to suggestions on whether we should keep it this way or not